website/package.json

```diff
@@ -67,15 +67,19 @@
     "@graphql-codegen/urql-introspection": "2.2.1",
     "@mizdra/graphql-codegen-typescript-fabbrica": "^0.6.0",
     "@monaco-editor/react": "4.6.0",
-    "@theguild/components": "6.6.3",
+    "@radix-ui/react-accordion": "^1.2.1",
+    "@radix-ui/react-icons": "^1.3.0",
+    "@theguild/components": "7.2.2",
     "classnames": "2.5.1",
     "date-fns": "2.30.0",
     "dedent": "1.5.3",
+    "eslint-plugin-tailwindcss": "^3.17.5",
```

Comment:
I guess the proper place will be in the root (since we use yarn v1 here we have a hoisting 💩)

Reply:
oh yeah, good catch